### PR TITLE
MGMT-18619: Obfuscate config ISO url

### PR DESCRIPTION
**Pull request by eranco74**
Use the ICI.UID instead of the ICI.name

### CODE REVIEW

File: controllers/imageclusterinstall_controller.go [hunk 1]
```diff
@@ -54,6 +53,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/sirupsen/logrus"
+
```
Comment (tsorya) [inline controllers/imageclusterinstall_controller.go:57]: delete space?

Comment (eranco74) [reply controllers/imageclusterinstall_controller.go:57]: God knows what goimport was doing :man_shrugging:

**Pull request merged**